docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -574,191 +574,34 @@ used to delimit variable names.
   doubling it. For example, to emit a header with the value ``100%``, the custom header value in
   the Envoy configuration must be ``100%%``.
 
-Supported variable names are:
+All HTTP :ref:`Command Operators <config_access_log_command_operators>` used for access logging may be specified
+in Custom request/response headers. However, depending where a particular Command Operator is used, the context needed for
+the Operator may not be available and the produced output is empty string. For example, the following configuration
+uses ``%RESPONSE_CODE%`` operator to modify request headers using code from the response.
+The output is an empty string, because request headers are modified
+before the request is sent upstream and the response is not received yet.
+
+.. literalinclude:: _include/header_formatters.yaml
+    :language: yaml
+    :linenos:
+    :lines: 15-20
+    :emphasize-lines: 3-6
+    :caption: :download:`header_formatters.yaml <_include/header_formatters.yaml>`
 
-%DOWNSTREAM_REMOTE_ADDRESS%
-    Remote address of the downstream connection. If the address is an IP address it includes both
-    address and port.
-
-    .. note::
-
-      This may not be the physical remote address of the peer if the address has been inferred from
-      :ref:`Proxy Protocol filter <config_listener_filters_proxy_protocol>` or :ref:`x-forwarded-for
-      <config_http_conn_man_headers_x-forwarded-for>`.
```

Comment:
nit: should we just keep these old docs for now and removed after the runtime guard is expired? 🤔